src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -219,10 +217,10 @@ function WorkspaceWorkflowsPage({policy, betas, route, reimbursementAccount, ses
                     </>
                 ),
                 isEndOptionRow: true,
-                isActive: policy?.reimbursementChoice !== CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_NO,
+                isActive: policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES,
```

Comment:
This prevents the toggle getting toggled on if a workspace has a non-USD currency (as the toggle will set it to `REIMBURSEMENT_MANUAL`). I think the previous commit worked fine for this.